draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -111,15 +112,13 @@ EDN:
 
 Verifiable Data Structure (VDS):
 
-: A data structure which supports one or more Proof Types.
-  This property is conceptually similar to "alg" (1), it described an algorithm used to maintain the verifiable data structure, for example a binary merkle tree algorithm.
+: A data structure which supports one or more Verifiable Data Structure Parameters.
+  This property describes an algorithm used to maintain a verifiable data structure, for example a binary merkle tree algorithm.
 
-Verifiable Data Structure Parameters (VDP):
+Verifiable Data structure Parameters (VDP):
```

Comment:
```suggestion
Verifiable Data Structure Parameters (VDP):
```

Paul, no.

Reply:
Can you explain why "VDP" is the proper acronym for "Verifiable Data [Ss]tructure Parameters" ?

Reply:
Acronym symmetry, mainly. VDSP seem needless intuitive. But I am not dying on that hill. I felt this was long resolved.

Reply:
It's consistently `Verifiable Data [Ss]tructure Proofs` now, e.g. https://github.com/cose-wg/draft-ietf-cose-merkle-tree-proofs/blob/main/draft-ietf-cose-merkle-tree-proofs.md#new-cose-header-parameters-param-list